src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -13,6 +13,7 @@
 
 use Psr\Log\LoggerInterface;
 use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\Chunk\LastChunk;
```

Comment:
oups
but for interop, we should never use instanceof on chunks, this should be isLast()

Reply:
fixed